packages/cosmo-eccodes-definitions/package.py

```diff
@@ -40,26 +41,29 @@ class CosmoEccodesDefinitions(Package):
     version('2.19.0.2', tag='v2.19.0.2')
     version('2.19.0.1', tag='v2.19.0.1')
     version('2.18.0.1', tag='v2.18.0.1')
-    version('2.14.1.2', tag='v2.14.1.2')
-    version('2.14.1.1', tag='v2.14.1.1')
 
     depends_on('eccodes')
```

Comment:
One abort in the logfile was that spack could not find the libs in eccodes.

Could be that spack is confused by the duplicate dep of eccodes, once through cosmo-eccodes-definitions, once through int2lm.
Especially since eccodes of the definitions is withouth `+fortran` variant.

Reply:
Works perfectly fine sequentially though.